package.json

```diff
@@ -17,7 +17,8 @@
     "stop:all": "npm stop; npm run stop:mockapi",
     "chimp": "NODE_ENV=acceptance chimp --mocha --browser=htmlunit --path=./test/acceptance/spec/",
     "chimp:dev": "NODE_ENV=acceptance chimp --mocha --path=./test/acceptance/ --watch --browser=firefox",
-    "lint": "eslint .",
+    "lint": "node_modules/.bin/eslint $FIXLINT .",
```

Comment:
Also, this more specific invocation of the executable avoids any possible nasties that could arise from the presence of duplicate globally installed entities (such as with the `npm -g` switch)